install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ $1 == "path" && $2 != "" ]]; then
+  version=$(curl -s -o- https://github.com/luizalabs/teresa/releases/latest | sed 's/.*tag\/\(.*\)\".*/\1/')
+
+  curl -L -O "https://github.com/luizalabs/teresa/releases/download/$version/teresa-linux-amd64"
+
+  chmod +x teresa-linux-amd64
+
+  mv teresa-linux-amd64 "$2/teresa"
```

Comment:
You may create a variable called `OUTPUT_PATH` that points to `/usr/local/bin/teresa` by default. If a path is provided, change it to `$1/teresa`. This way, you conditional will only need to handle the path, and not the download. This way, you won't have to repeat the same code for downloading it.

Reply:
@vinicyusmacedo The only problem with this way is because of the different messages for different scenarios. I can create a function to abstract the chmod + download, what do you think?

Reply:
I did two ways, to be possible to install without `sudo` in a specified path

Reply:
I get it @piiih . In that case, you could instead keep `$1` on a variable `PROVIDED_PATH` and, if it is provided, you set `OUTPUT_PATH=$PROVIDED_PATH/teresa` and also use it to check if sudo is needed (if `PROVIDED_PATH` is null, exit 1 and ask for sudo).